src/main/resources/templates/index.html

```diff
@@ -77,8 +77,10 @@
       </div>
     </div>
   </main>
-  <tamufooter></tamufooter>
+  <tl-footer></tl-footer>
+  <tl-themes active-theme="tamu"></tl-themes>
 
+  <script src="https://api.library.tamu.edu/tl-components/2x/tl-components.js"></script>
```

Comment:
Should we let the browser select between http and https protocol?

Reply:
Modfied the URL as per the suggestion